### ISSUE TITLE
Compatibility with uglify plugin?

### ISSUE THREAD

**Issue by alexhancock** [webpack-contrib/script-loader#9]
Is there a known issue with comments not being uglified out of modules being loaded with `script-loader`?

For instance, I have an entry file which looks like this...

``` coffeescript
// vendor.coffee
require("script!jquery/dist/jquery.min.js")
require("script!../assets/bootstrap/js/bootstrap.js")
```

...and a simplified version of my webpack config looks like this...

``` coffeescript
config =
  entry:
    vendor: "./vendor.coffee"
  plugins: [
    new webpack.optimize.UglifyJsPlugin({ output: {comments: false} }) // *
  ]
```

\*  using temporary solution for the structure of the `comments` output option from [here](https://github.com/webpack/webpack/issues/324#issuecomment-46957483)

In this case, the resulting file still has a bunch of comments in it.

Am I missing something, or does uglify comment stripping just not work with files loaded with the `script-loader`? Are there other limitation when it comes to minification I should be aware of?

Thanks in advance for any help!

**Comment by gabrielhpugliese** [comment]
I have the exactly same issue. Was trying to change my build from gulp but the result becomes much bigger.

**Comment by alexhancock** [comment]
I understand now, at least in part why it doesn't work. Given that the content of the files loaded with a script loader is just taken and wrapped in an eval.

I followed the advice [here](https://github.com/webpack/script-loader/issues/3) and wrote a simpler "uglify only" loader, which I will at least put on npm in case it's useful to someone else.

I also found [this loader](https://github.com/bestander/uglify-loader) which I am experimenting with to see if it fits my needs.

I am trying to get source maps working, which I think the loader I just referenced achieves.
